libfreerdp/primitives/primitives.c

```diff
@@ -247,7 +247,7 @@ static BOOL primitives_autodetect_best(primitives_t* prims)
 			goto out;
 		}
 
-		WLog_DBG(TAG, " * %s\t= %" PRIu32, cur->name, cur->count);
+		WLog_DBG(TAG, " * %s= %" PRIu32, cur->name, cur->count);
```

Comment:
I´ve run into issues with the performance test.
On one of my machines opencl did not initialize and the whole test is failing the session.
Could you make this fail silent?

Reply:
ok will look at that.